src/main/java/com/google/sps/data/friend_map/UserNode.java

```diff
@@ -15,33 +15,36 @@
 package com.google.sps.data.friend_map;
 
 import java.util.Collection;
+import javax.annotation.Nonnull;
 import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.base.Preconditions;
 
 /**
 * Class to represent a User in the map representing all user friendships
 */
 public class UserNode {
 
   private final String userID;
-  private ImmutableSet<String> currentUserFriendIDs;
+  private final ImmutableSet<String> friendIDs;
 
   /**
-  * Constructs an instance of a UserNode
+  * Creates a node to represent a user with their set of friends
```

Comment:
Verbage - drop the "c" ("Create a node...")

Reply:
Got it!